src/appkit.cs

```diff
@@ -27689,4 +27861,147 @@ interface NSSharingServicePickerToolbarItemDelegate
+
+		[Export ("snapshot")]
+		NSDiffableDataSourceSnapshot<NSObject, NSObject> Snapshot ();
+
+		[Export ("applySnapshot:animatingDifferences:")]
+		void ApplySnapshot (NSDiffableDataSourceSnapshot<NSObject, NSObject> snapshot, bool animatingDifferences);
+
+		[Export ("applySnapshot:animatingDifferences:completion:")]
+		void ApplySnapshot (NSDiffableDataSourceSnapshot<NSObject, NSObject> snapshot, bool animatingDifferences, [NullAllowed] Action completion);
+
+		[Export ("itemIdentifierForRow:")]
+		[return: NullAllowed]
+		NSObject ItemIdentifierForRow (nint row);
+
+		[Export ("rowForItemIdentifier:")]
+		nint RowForItemIdentifier (NSObject identifier);
+
+		[Export ("sectionIdentifierForRow:")]
+		[return: NullAllowed]
+		NSObject SectionIdentifierForRow (nint row);
+
+		[Export ("rowForSectionIdentifier:")]
+		nint RowForSectionIdentifier (NSObject identifier);
+
+		[NullAllowed, Export ("rowViewProvider", ArgumentSemantic.Copy)]
+		NSTableViewDiffableDataSourceRowProvider RowViewProvider { get; set; }
+
+		[NullAllowed, Export ("sectionHeaderViewProvider", ArgumentSemantic.Copy)]
+		NSTableViewDiffableDataSourceSectionHeaderViewProvider SectionHeaderViewProvider { get; set; }
+
+		[Export ("defaultRowAnimation", ArgumentSemantic.Assign)]
+		NSTableViewAnimationOptions DefaultRowAnimation { get; set; }
+	}
+
+	[Protocol]
+	interface NSTextContent
+	{
+		[Mac (10, 16)]
+		[Abstract]
+		[Export ("contentType")]
+		NSString GetContentType ();
+
+		[Mac (10, 16)]
+		[Abstract]
+		[Export ("setContentType:")]
+		void SetContentType (NSString contentType);
+	}
+
+
+	[Mac (10, 16)]
+	enum NSTextContentType {
+		[Field ("NSTextContentTypeUsername")]
+		NSTextContentTypeUsername,
+
+		[Field ("NSTextContentTypePassword")]
+		NSTextContentTypePassword,
+
+		[Field ("NSTextContentTypeOneTimeCode")]
+		NSTextContentTypeOneTimeCode,
+	}
+
+	[Mac (10,16)]
+	[BaseType (typeof(NSObject))]
+	interface NSTintConfiguration : NSCopying, NSSecureCoding
+	{
+		[Static]
+		[Export ("defaultTintConfiguration", ArgumentSemantic.Strong)]
+		NSTintConfiguration DefaultTintConfiguration { get; }
+
+		[Static]
+		[Export ("monochromeTintConfiguration", ArgumentSemantic.Strong)]
+		NSTintConfiguration MonochromeTintConfiguration { get; }
+
+		[Static]
+		[Export ("tintConfigurationWithPreferredColor:")]
+		NSTintConfiguration TintConfigurationWithPreferredColor (NSColor color);
+
+		[Static]
+		[Export ("tintConfigurationWithFixedColor:")]
+		NSTintConfiguration TintConfigurationWithFixedColor (NSColor color);
+
+		[NullAllowed, Export ("baseTintColor", ArgumentSemantic.Strong)]
+		NSColor BaseTintColor { get; }
+
+		[NullAllowed, Export ("equivalentContentTintColor", ArgumentSemantic.Strong)]
+		NSColor EquivalentContentTintColor { get; }
+
+		[Export ("adaptsToUserAccentColor")]
+		bool AdaptsToUserAccentColor { get; }
+	}
+
+	[Mac (10,16)]
+	[BaseType (typeof(NSToolbarItem))]
```

Comment:
add `[DisableDefaultCtor]` since there's no way to set `itemIdentifier` after creation and it's `null` is not accepted in the designated initializer